proxy/http2/Http2Stream.cc

```diff
@@ -509,7 +509,7 @@ Http2Stream::update_write_request(IOBufferReader *buf_reader, int64_t write_len,
         parent->connection_state.send_headers_frame(this);
 
         // See if the response is chunked.  Set up the dechunking logic if it is
-        is_done = this->response_initialize_data_handling();
+        this->response_initialize_data_handling();
```

Comment:
Should you check or assert on the return value here? I would guess that bad things happen if ``response_initialize_data_handling`` returns false.